auto_verify.sh

```
@@ -82,7 +82,56 @@ run_verification() {
         fi
     fi
 
+    #install docker
+    if [ "$install_docker" == "yes" ]; then
```

Shellcheck (0.10.0)

[warning] install_docker is referenced but not assigned.

(SC2154)